src/hipercow/cli.py

```diff
@@ -123,23 +123,23 @@ def cli_driver_configure(name: str):
     #
     # For now, let's just have it on/off.
     r = root.open_root()
-    configure(r, name)
+    configure(name, root=r)
 
 
 @driver.command("unconfigure")
 @click.argument("name")
 def cli_driver_unconfigure(name: str):
     """Unconfigure (remove) a driver."""
     r = root.open_root()
-    unconfigure(r, name)
+    unconfigure(name, r)
 
 
 @driver.command("show")
 @click.argument("name", required=False)
 def cli_driver_show(name: str | None):
     """Show configuration for a driver."""
     r = root.open_root()
-    show_configuration(r, name)
+    show_configuration(name, root=r)
```

Comment:
I don't think this one needs the `root=`   ? (Not that it matters)

`configure.py:34:def show_configuration(driver: str | None, root: Root) -> None:`